.gitattributes

```diff
@@ -0,0 +1,4 @@
+*.js text eol=lf
+*.cjs text eol=lf
+*.ts text eol=lf
+*.mjs text eol=lf
```

Comment:
Please add these settings to your local git config, not to the repository.

Reply:
well, its the repos prettier config which causes it to error on every single line with the default git settings on windows (convert LF to CRLF on pull, and back to LF on push)

i can definitely remove it, but i'd urge you to consider changing *something* so the default behaviour on windows when using your recommended development setup isn't *errors on every single line*

Reply:
A note in `DEVELOPING.md`, describing how to configure git locally, would be the preferred alternative. Because that's missing in our recommended development setup.

Reply:
aight, if thats what you want, i'll remove the commit. its not ideal though as this is *not* something which should be configured globally. And you can't configure it locally before cloning, so you'll have to:
- clone
- configure
- git rm everything
- git reset --hard
to fix the line endings. which is quite an annoyingly convoluted process for something like line endings which usually just works (as most projects don't care about having CRLF during development)